app/schemas.py

```diff
@@ -0,0 +1,170 @@
+from typing import Optional
+from uuid import UUID, uuid4
+from datetime import datetime, timezone
+from pydantic import BaseModel, EmailStr, Field, condecimal, PositiveInt
+from fastapi import Query
+from decimal import Decimal
+from app.api.routes.dependencies import get_current_time
+
+
+
+
+
+password_regex = r"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&#])[A-Za-z\d@$!%*?&#]{8,}$" 
+
+class Status:
```

Comment:
Remind me, why do we have this class?

Reply:
This is just what was approved in a previous PR for my partner, and as i mention in this PR details it's only for setting up the database and structural matters, my partner and me will distribute tasks to completely rebuild the project to work efficiently with the database. Please dont judge now.

Reply:
It was a question to see if we still need it or not. In that case, just remind @Omarmanassrah to take care of this part in his changes.